totest-manager.py

```diff
@@ -257,17 +274,28 @@ def overall_result(self, snapshot):
                         to_ignore = True
                 # to_ignore can happen with or without refs
                 ignored = True if to_ignore else len(refs) > 0
+                build_nr = str(job['settings']['BUILD'])
                 for ref in refs:
                     if ref not in self.issues_to_ignore:
                         if to_ignore:
-                            self.issues_to_ignore.append(ref)
+                            self.issues_to_ignore[ref] = build_nr
                             self.update_pinned_descr = True
-                            with open(self.issuefile, 'a') as f:
-                                f.write("%s\n" % ref)
                         else:
                             ignored = False
+                    else:
+                        # update reference
+                        self.issues_to_ignore[ref] = build_nr
 
-                if not ignored:
+                if ignored:
+                    if labeled:
+                        text = 'Ignored issue' if len(refs) > 0 else 'Ignored failure'
+                        # remove flag - unfortunately can't delete comment unless admin
+                        data = {'text': text}
+                        self.openqa.openqa_request(
+                            'PUT', 'jobs/%s/comments/%d' % (job['id'], labeled), data=data)
+
+                    logger.info("job %s failed, but was ignored", jobname)
```

Comment:
Could you extend this to "job %s failed, but was ignored due to '@ttm ignore' comment in job" to be explicit?

Reply:
we don't know at this point if the comment contained any action - or if this was a carry over who lead to the ignore. So no.